Jenkinsfile

```diff
@@ -1 +1,4 @@
-buildPlugin(useContainerAgent: true, failFast: false, platforms: ['linux'])
+buildPlugin(useContainerAgent: true, failFast: false, configurations: [
+  [platform: 'linux', jdk: 17],
+  [platform: 'windows', jdk: 11],
```

Comment:
The default Java version for Linux is Java 8, which no longer works on recent Jenkins cores. This change follows the recommended configuration in the archetype by adding a Linux Java 17 build and a Windows Java 11 build.